code/__DEFINES/combat.dm

```diff
@@ -41,6 +41,8 @@
 #define MAGIC "magic"
 /// Involved in checking the likelyhood of applying a wound to a mob.
 #define WOUND "wound"
+/// Involves being consumed
+#define VORE "vore"
```

Comment:
dont name it VORE name it something like moth_eating dummy

Reply:
or consume

Reply:
I have a mouth and I must feed